packages/aws-cdk/lib/init-templates/app/util/os.ts

```diff
@@ -24,30 +22,26 @@ export async function shell(command: string[], options: ShellOptions = {}): Prom
 
     // Both write to stdout and collect
     child.stdout.on('data', chunk => {
-      if (!options.quiet) {
```

Comment:
Why did the `quiet` feature get removed?

Reply:
(If it's because it was unused and you took the opportunity to clean up a bit, please describe that in the PR so reviewers don't have to guess. I usually add an "Also in this PR..." section for those)